### DISCUSSION TITLE
Next Translate Error on Safari Device

### DISCUSSION THREAD

**Discussion by joaobibiano** [vercel/next.js#22110] [Help]
**What example does this report relate to?**

next-translate

**What version of Next.js are you using?**

10.0.6

**What version of Node.js are you using?**

v15.8.0

**What browser are you using?**

Safari

**What operating system are you using?**

IOS

**How are you deploying your application?**

Vercel

**Describe the Bug**

When oppening my website on Safari Device, it throw an error:

Unhandled Runtime Error

TypeError: undefined is not a constructor (evaluating 'new Intl.PluralRules (lang)' )

**Expected Behavior**

Should not throws any error

**To Reproduce**

Just open any website with next-translate configured on a IOS with Safari Browser

![photo5837006245305890589](https://user-images.githubusercontent.com/29175815/107154891-38c94b80-696d-11eb-8bc8-07a1f0482f14.jpg)

![photo5837006245305890590](https://user-images.githubusercontent.com/29175815/107154893-3bc43c00-696d-11eb-859d-41e078e7ff15.jpg)

**Comment by ghost** [comment]
This issue doesn't belong here and should be closed. Please, report it to [next-translate](https://github.com/vinissimus/next-translate/issues).

`Intl` is a very good standard library, but not all browsers offer full functionality for it...
